### PR TITLE
Feature/user path hashing

### PR DESCRIPTION
**Pull request by sontiveros-AT**
Refactor of file_manager, models and orm's

### CODE REVIEW

File: commons/file_manager.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    @staticmethod
+    def create_project(project_name, description, language_id, user_id):
+        user_dir = OrmUser.get_user_dir(user_id)
+        language = OrmLanguage.get_language(language_id)
+        project_path = 'media' / Path(user_dir) / language.name / \
+            language.version / project_name
+
+        # create project in the database and sends the id
+        print(FileManager.is_repeated_project(
+            project_name, language_id, user_id))
```
Comment (andres-cox) [inline commons/file_manager.py]: print statement

File: code_editor/views/project_view.py [hunk 2]
```diff
@@ -38,43 +30,26 @@ class ProjectView(TemplateView):
...
 
-        # creates file and adds it to the database
-        file = FileManager()
-        file_name = ''
-        if language_name == 'python':
-            file_name = 'main'
-        elif language_name == 'java':
-            file_name = 'Main'
+        if project:
+            return redirect('/api/v1/project/{}'.format(project.id))
```
Comment (andres-cox) [inline code_editor/views/project_view.py:48]: este if es necesario?

File: commons/file_manager.py (hunk 1 above)
Comment (sontiveros-AT) [reply commons/file_manager.py]: I deleted print statements in the last commit

File: code_editor/views/project_view.py (hunk 2 above)
Comment (sontiveros-AT) [reply code_editor/views/project_view.py:48]: It checks if project is an object, if it is None it continues and sends a message that tells the user the project already exists.

**Review by andres-cox — APPROVED** [review]
Nice update, nothing to comment

**Pull request merged**